src/function/table/system/duckdb_functions.cpp

```diff
@@ -259,6 +269,18 @@ struct MacroExtractor {
 		return Value::LIST(LogicalType::VARCHAR, std::move(results));
 	}
 
+	static vector<LogicalType> GetParameterLogicalTypes(ScalarMacroCatalogEntry &entry, idx_t offset) {
+		vector<LogicalType> results;
+		auto &macro_entry = *entry.macros[offset];
+		for (idx_t i = 0; i < macro_entry.parameters.size(); i++) {
+			results.emplace_back(LogicalType::VARCHAR);
+		}
+		for (idx_t i = 0; i < macro_entry.default_parameters.size(); i++) {
+			results.emplace_back(LogicalType::VARCHAR);
+		}
+		return results;
```

Comment:
I think we should default to `LogicalType::UNKNOWN` for macros here, but not sure.

Reply:
done